pkg/kudoctl/cmd/init/manager.go

```diff
@@ -59,19 +59,19 @@ func NewOptions(v string) Options {
 // Install uses Kubernetes client to install KUDO.
 func Install(client *kube.Client, opts Options, crdOnly bool) error {
 
-	clog.V(4).Printf("installing crds")
+	clog.Printf("✓ installing crds")
 	if err := installCrds(client.ExtClient); err != nil {
 		return err
 	}
```

Comment:
Now, that I think about it, it should be the other way around ;)
```suggestion
	if err := installCrds(client.ExtClient); err != nil {
		return fmt.Errorf("❌failed to install CRDs: %w", err)
	}
	clog.Printf("✓ installed crds")
```

Reply:
and the others too ;)

Reply:
I prefer @zen-dog graphics... and I do not like graphics in the error messages

Reply:
🤔 so do we want to add the cross there or not? I am not sure I understand the conclusion of that sentence